packages/samples/mvp-dapp/src/views/load.view.ts

```diff
@@ -0,0 +1,60 @@
+// import type { StoreType } from '../record/record.model'
+
+type ItemData = {
+  key: string
+  value: string
+  label: string
+}
+
+export type StoreType = {
+  data: {
+    profile: ItemData[]
+    addresses: ItemData[]
+    custom: ItemData[]
+    dweb: ItemData[]
+  }
+}
+
+export class Load {
+  static toJson(storage: StoreType): string {
```

Comment:
The method name indicates a `JSON` is returned, but a string is returned in fact.